### DISCUSSION TITLE
Import airtable to sql database issue [May 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2265] [Discord Threads]
> 🙋‍♂️ **Mr.Raccoon**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### can you please paste project info here  
#  
> 💁 **o1lab**  
#### (right top menu)  
#  
> 🙋‍♂️ **Mr.Raccoon**  
#### Node: **v16.14.2**   
#### Arch: **x64**   
#### Platform: **win32**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.91.0**  
#  
> 💁 **o1lab**  
#### after the error, close the modal  
#  
> 💁 **o1lab**  
#### and see if you can see tables and data  
#  
> 🙋‍♂️ **Mr.Raccoon**  
#### i see tables and cloumns but dont see data  
#  
> 💁 **o1lab**  
#### ok  
#  
> 💁 **o1lab**  
#### will DM you  
#  
> 💁 **o1lab**  
#### Please try the latest release v0.91.1   
####    
#### We've made improvements  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/979273637355995186)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
